.github/workflows/cookiecutter-e2e.yml

```diff
@@ -21,8 +24,13 @@ jobs:
       fail-fast: true
       matrix:
         include:
+          - { cookiecutter: "tap-template", replay: "tap-graphql-jwt.json", python-version: "3.9",  os: "ubuntu-latest" }
           - { cookiecutter: "tap-template", replay: "tap-rest-api_key-github.json", python-version: "3.9",  os: "ubuntu-latest" }
-          - { cookiecutter: "tap-template", replay: "tap-rest-jwt-github.json", python-version: "3.9",  os: "ubuntu-latest" }
```

Comment:
You might wanna delete the corresponding replay file too

Reply:
diff display is a bit confusing here - the file was renamed `tap-rest-jwt.json` and is included three lines down

Reply:
Weird, that's the intent I imagined, but I'm seeing the file when I checkout your branch:

```console
$ tree e2e-tests/cookiecutters
e2e-tests/cookiecutters
├── README.md
├── tap-graphql-jwt.json
├── tap-rest-api_key-github.json
├── tap-rest-basic_auth.json
├── tap-rest-bearer_token.json
├── tap-rest-custom.json
├── tap-rest-jwt-github.json
├── tap-rest-jwt.json
├── tap-rest-oauth2.json
├── target-per_record.json
└── test_cookiecutter.sh

1 directory, 11 files
```